test/e2e/tests/account/create-remove-account-snap.spec.ts

```diff
@@ -33,7 +33,8 @@ describe('Create and remove Snap Account', function (this: Suite) {
           WINDOW_TITLES.ExtensionInFullScreenView,
         );
         const headerNavbar = new HeaderNavbar(driver);
-        await headerNavbar.checkAccountLabel('SSK Account');
+        // BUG #37591 - With BIP44 the account mame is not retained.
+        await headerNavbar.checkAccountLabel('Snap Account 1');
```

Comment:
To be reworked in a follow-up PR: This is true, and IDK if we should consider this a bug or not (since we're doing a some rework in the Snap account flows). To be checked!

> [!NOTE]
> We should check other mentions of this bug in all E2E too!